### DISCUSSION TITLE
Feature Request: Add ability to run multiple patched Reddit apps on one device

### DISCUSSION THREAD

**Discussion by wibuf** [ReVanced/revanced-discussions#990] [General]
I don't believe this is a feature yet, I've done quite a bit of searching. I would like to use two third party apps, (RiF and Baconreader), with the OAuth Client ID patches on the same device. 

In the current reddit_client_id_revanced.txt, we can only specify a single client ID. Is there any way to make that a csv, or use a second reddit_client_id_revanced.txt to specify a second application?

Thanks

**Comment by KobeW50** [comment]
The txt file is not tied to the patched application. You only need it at the time of patching. After you patch and install one, you can change the txt file and patch and install the other app. You can delete the txt file after if you want
